src/hypomnema/api/helpers.py

```
@@ -451,5 +545,45 @@ def create_sub(
+    return []
+  content = [content] if isinstance(content, str) else content
+  result: list[Any] = []
+  for item in content:
+    match item:
+      case str():
+        result.append(item)
+      case SubLike():
+        if sub_only:
+          result.append(item)
+        else:
+          raise TypeError(f"Invalid item, expected str or SubLike, got {type(item)}")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)